test/debugger/test-debug-break-on-uncaught.js

```diff
@@ -91,7 +91,10 @@ function runScenario(scriptName, throwsOnLine, next) {
 
         client.reqContinue(function(error) {
           assert.ifError(error);
-          setTimeout(assertHasPaused.bind(null, client), 100);
+          setTimeout(
+            assertHasPaused.bind(null, client),
+            common.platformTimeout(3000)
+          );
```

Comment:
/cc @nodejs/testing 

Might it be better to use a short `setInterval()` (or a `setTimeout()` that renews itself if the thing hasn't paused yet) and let the test time out via `test.py` rather than having an `AssertionError` here?